PennMobile/src/main/java/com/pennapps/labs/pennmobile/GsrFragment.kt

```diff
@@ -0,0 +1,519 @@
+package com.pennapps.labs.pennmobile
+
+import android.app.DatePickerDialog
+import android.app.TimePickerDialog
+import android.os.Bundle
+import android.support.v4.app.Fragment
+import android.support.v7.widget.LinearLayoutManager
+import org.joda.time.format.DateTimeFormat
+import com.pennapps.labs.pennmobile.classes.GSRContainer
+import com.pennapps.labs.pennmobile.classes.GSRLocation
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.*
+import com.pennapps.labs.pennmobile.api.Labs
+import com.crashlytics.android.Crashlytics
+import com.crashlytics.android.answers.Answers
+import com.crashlytics.android.answers.ContentViewEvent
+import com.pennapps.labs.pennmobile.R.string.gsr
+import com.pennapps.labs.pennmobile.classes.GSR
+import io.fabric.sdk.android.Fabric
+import kotlinx.android.synthetic.main.fragment_gsr.view.*
+import org.joda.time.DateTime
+import org.joda.time.DateTimeZone
+import java.util.*
+import java.util.concurrent.TimeUnit
+
+
+class GsrFragment : Fragment() {
+
+    // ui components
+    var calendarButton :Button? = null
```

Comment:
For UI components you could use lateinit var, and make it nonnullable. Also the colon spacing here is slightly different from the other lines.